Makefile

```diff
@@ -30,6 +54,10 @@ generate:
 .PHONY: build
 build: go-build
 
+.PHONY: build-linux-static
```

Comment:
[nit] I think I'd rather drop the `-static`. Let's not repeat past mistakes -- there's one binary for each platform, let's try to keep it at that 😅

Reply:
Ok removed.